src/NIST-800-53/rules/s3/nist80053S3BucketPublicWriteProhibited.ts

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { CfnBucket } from '@aws-cdk/aws-s3';
+import { CfnResource } from '@aws-cdk/core';
+
+/**
+ * S3 Buckets prohibit public write access through their Block Public Access configurations and bucket ACLs - (Control IDs: AC-3, AC-4, AC-6, AC-21(b), SC-7, SC-7(3))
+ * @param node the CfnResource to check
+ */
+export default function (node: CfnResource): boolean {
```

Comment:
The rule needs to check a combination of the following
   - The Block Public Access setting restricts public policies.
   - The Block Public Access setting restricts public ACLs or the bucket ACL does not allow public write access